src/util/languageUtils.js

```diff
@@ -0,0 +1,47 @@
+import url from 'url';
+import Accepts from 'accepts';
+
+export const LANG_DEFAULT = 'en-US';
+
+export const getLanguage = (request, supportedLangs, defaultLang=LANG_DEFAULT) => {
+	const firstPathComponent = request.url.path.split('/')[1];
+	const urlLang = supportedLangs.includes(firstPathComponent) ? firstPathComponent : null;
+	const browserLang = Accepts(request).language(supportedLangs);
+
+	const langSourcePreference = [
```

Comment:
yup, correct - there's a sequence of about 3-4 tasks to get this fully up to spec - member profile language support will be done in [WP-122](https://meetup.atlassian.net/browse/WP-122)